.changeset/sheet-shock-round.md

```diff
@@ -0,0 +1,7 @@
+---
+"@utrecht/mapcontrolbutton}-css": major
```

Comment:
```suggestion
"@utrecht/map-control-button-css": major
```

Reply:
Ah te veel afgegaan op de naam van het mapje in VS code :)